packer_test/core_tests/local_eval_test.go

```diff
@@ -12,20 +16,21 @@ func (ts *PackerTestSuite) TestEvalLocalsOrder() {
 		Runs(10).
 		Stdin("local.test_local\n").
 		SetArgs("console", "./templates/locals_no_order.pkr.hcl").
-		Assert(MustSucceed(), Grep("\\[\\]", grepStdout, grepInvert))
+		Assert(lib.MustSucceed(),
+			lib.Grep("\\[\\]", lib.GrepStdout, lib.GrepInvert))
 }
 
-func (ts *PackerTestSuite) TestLocalDuplicates() {
+func (ts *PackerCoreTestSuite) TestLocalDuplicates() {
 	pluginDir, cleanup := ts.MakePluginDir()
 	defer cleanup()
 
 	for _, cmd := range []string{"console", "validate", "build"} {
 		ts.Run(fmt.Sprintf("duplicate local detection with %s command - expect error", cmd), func() {
 			ts.PackerCommand().UsePluginDir(pluginDir).
 				SetArgs(cmd, "./templates/locals_duplicate.pkr.hcl").
-				Assert(MustFail(),
-					Grep("Duplicate local definition"),
-					Grep("Local variable \"test\" is defined twice"))
+				Assert(lib.MustFail(),
```

Comment:
My first question here is what is lib? Why not just `MustFail()`. The generic name lib gives no information on the relation of MustFail() to the test suite. 

What do you think  if we renamed Assert to `RunAndAssert` to make it clear that Assert runs the test suite and succeeds if the asserts are true? 

```go
RunAndAssert(
  MustFail(), 
  Grep("Duplicate local definition"), 
  Grep("Local variable \"test\" is defined twice"))
```